src/features/Consumer/components/InsightDisplay.module.scss

```
@@ -0,0 +1,16 @@
+.stat {
```

Rmb to follow the class name convention!
```suggestion
.InsightDisplay__Stat {
```